src/executor/index.ts

```diff
@@ -113,16 +113,20 @@ async function executor (
   const db = createDatabase(config.extensions)
 
   const usedTables = extractTables(sql)
```

Comment:
What do you think of:

```suggestion
  const ast = parseSql(sql)
  const usedTables = extractTables(ast) # We need to change this method contract
```

This way we have one parse only 🤔

Reply:
done!